server/handlers/captureHandler.js

```diff
@@ -94,7 +96,38 @@ const captureHandlerPost = async function (req, res) {
   }
 };
 
+const captureHandlerPatch = async function (req, res) {
+  const { capture_id } = req.params;
+  const session = new Session();
+  const captureRepo = new CaptureRepository(session);
+  const executeUpdateCapture = updateCapture(captureRepo);
+  const updateSchema = Joi.object({
+    id: Joi.any().forbidden(),
+    lat: Joi.any().forbidden(),
+    lon: Joi.any().forbidden(),
+    location: Joi.any().forbidden(),
+    created_at: Joi.any().forbidden(),
+  });
+  try {
+    const value = await updateSchema.unknown(true).validateAsync(req.body, {
```

Comment:
This line can be outside the try block. There is a global error handler for this app that catches the validation error thrown by joi. Please see `server/handlers/utils.js` and it is being used in `app.js`.